### PR TITLE
Synergize site with social

### PR DESCRIPTION
**Pull request by greatbacon**
This PR takes another pass at the website with the primary goal of getting it https://felt.social in-line from a style & content perspective.

paired with https://github.com/feltcoop/felt.social/pull/54

I copied over a handful of components (😖) but I wasn't 100% sure what should/could be upstreamed & I wanted to ride this wave as long as I could. Sorry!

Anyway, this PR is the meatiest of the 2 and does the following:
- The big thing is that I moved the `about` page from social to this site. Since this repo is the source of truth on our values & mission, this seemed the like the best place for the project about page to live.
- I also did some styling and content tweaking on the about page while I was at it.
- Ported the `SocialLink` footer to match felt.social. I only have it pointing to the feltjs github though, wasn't sure if we wanted the co-op socials too?
- Brought over the Header component to match sizing & logo
- Pulled the Repos & Values out into their own components

### CODE REVIEW

File: src/lib/project.ts [hunk 1]
```diff
@@ -22,8 +22,7 @@ export interface Operator {
 export const project: Project = {
 	name: 'Felt',
 	source: 'https://github.com/feltjs/felt',
-	summary:
-		'Felt is free and open source software and tools for building and maintaining communities, publishing to the web, and collaborating with information💚',
+	summary: 'A free and open source toolkit for building and maintaining communities',
```
Comment (greatbacon) [inline src/lib/project.ts:25]: I shortened the summary to make it punchier, with a more verbose description now living in the Intro component.

File: src/routes/about/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+		<p>
+			On top of that toolkit we are building the enterprise
+			<a href="https://www.felt.social/">Felt.social</a>
+			(@feltcoop on
+			<a href="https://github.com/feltcoop">GitHub</a>,
+			<a href="https://twitter.com/feltcoop">Twitter</a>, and
+			<a href="https://fosstodon.org/@feltcoop">Mastodon</a>). It is planned as a collectively-owned
+			platform, free of ads and investors, that supports & hosts communities that are human-scale,
+			self-governed, consentful, private by default, and deeply customizable and programmable.
+		</p>
```
Comment (ryanatkn) [inline src/routes/about/+page.svelte:33]: I don't think this belongs here

File: src/routes/about/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+			ownership and control. We believe this model will help Felt The Business remain
+			<span class="font-bold whitespace-no-wrap">→ accountable ←</span>
+			to its users, workers, and society. To learn more about why we chose this business structure, see
+			our (forthcoming) blog post "Why Felt is a worker co-op". In the future, we want to explore the
+			<a href="https://platform.coop/">platform cooperative model</a>
+			to share control and ownership with our users. (you, we hope!) Our
+			<a href="https://github.com/feltjs/felt/blob/main/GOVERNANCE.md">GOVERNANCE.md</a> reflects this
+			goal while being up front about our current dictator status as workers.
+		</p>
+	</section>
```
Comment (ryanatkn) [inline src/routes/about/+page.svelte:99]:
```suggestion
```

belongs on Felt.social and should probably be reworked there

File: src/routes/about/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+			</p>
+		</details>
+		<details>
+			<summary> Why isn't Felt decentralized/federated? </summary>
+			<p>
+				We expect our tech — which is focused on <em>communities</em>, not <em>messaging</em> — to
+				eventually be decentralized/federated/etc, perhaps with ActivityPub or Matrix or both, but
+				today we are focused on UX quality and implementation velocity. We find it much easier to
+				deliver good UX with a centralized architecture given the nature of decentralized systems
+				and our small self-funded team with no expertise in such things. If Matrix fits your
```
Comment (ryanatkn) [inline src/routes/about/+page.svelte:134]: I think a paragraph break before "If Matrix" would be good here

File: src/routes/about/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+				usecases, we recommend considering using it instead of Felt. For microblogging, we recommend
+				Mastodon. Felt's focus is on communities and their custom needs, with an emphasis on
```
Comment (ryanatkn) [inline src/routes/about/+page.svelte:136]: I think a paragraph break before "Felt's focus" would be good here

File: src/routes/about/+page.svelte [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+				attempting to follow the standard ActivityPub/Fediverse/Mastodon/etc
+				<a href="https://www.w3.org/TR/activitystreams-vocabulary/">ActivityStreams vocabulary</a>,
+				(but not ActivityPub the federated protocol, not yet) setting us up for potential future
+				compatibility. We're also maintaining JSON schemas that can be used in any programming
+				language, so at least our off-spec stuff is machine-readable.
+			</p>
+		</details>
+
+		<details>
+			<summary>What features will Felt have?</summary>
```
Comment (ryanatkn) [inline src/routes/about/+page.svelte:147]: Looking at this now seems we should talk about what we concretely have, not show a wishlist, and then link to the roadmap. And it veers away from features near the end. Cutting for now seems good

File: src/routes/about/+page.svelte [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+			</ul>
+		</details>
+		<details>
+			<summary>How much does Felt cost?</summary>
+			<p>
+				If you want to self host, you can go spin up your own instance today. If that sounds like a
+				paint though, hosting & memberships will be some modest amount of money, hopefully less than
+				you think. We'll probably have a way to invite friends for free. Check back in Q1 2023.
+			</p>
+		</details>
```
Comment (ryanatkn) [inline src/routes/about/+page.svelte:175]:
```suggestion
```

belongs in Felt.social

File: src/routes/about/+page.svelte [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+				problem. Matrix prioritizes privacy and security in a federated context; we prioritize UX in
+				a centralized context. What goes for Mastodon goes for Felt: don't send anything in DMs that
+				you don't trust the admin with, or anything sensitive at all.
+			</p>
+		</details>
+		<details>
+			<summary> Why isn't Felt decentralized/federated? </summary>
+			<p>
+				We expect our tech — which is focused on <em>communities</em>, not <em>messaging</em> — to
+				eventually be decentralized/federated/etc, perhaps with ActivityPub or Matrix or both, but
```
Comment (ryanatkn) [inline src/routes/about/+page.svelte:131]:
```suggestion
				eventually support a decentralized protocol, perhaps with ActivityPub or Matrix or both, but
```

File: src/lib/Intro.svelte [hunk 9]
```diff
@@ -0,0 +1,7 @@
+<p class="panel padded-xl">
+	<strong>Felt</strong>
+	is a
+	<small><code>work in progress</code></small> toolkit of free and open source software. In accordance
+	with our mission to make our lives better with technology, we seek to build software tools that make
```
Comment (ryanatkn) [inline src/lib/Intro.svelte:5]: seems we could drop "accordance with"

File: src/lib/Intro.svelte [hunk 10]
```diff
@@ -0,0 +1,7 @@
+<p class="panel padded-xl">
+	<strong>Felt</strong>
+	is a
+	<small><code>work in progress</code></small> toolkit of free and open source software. In accordance
```
Comment (ryanatkn) [inline src/lib/Intro.svelte:4]: seems we could drop "accordance with"

File: src/lib/Intro.svelte [hunk 11]
```diff
@@ -0,0 +1,7 @@
...
+	with our mission to make our lives better with technology, we seek to build software tools that make
+	it simple and easy to maintain communities, publish to the web, and collaborate with information 💚
```
Comment (ryanatkn) [inline src/lib/Intro.svelte:6]: could rework this in a number of ways, feels a bit wordy and awkward, I think "build and maintain communities" should be kept together, and the "build" before that could be changed to "make" or something else

maybe "seek" could maybe be cut, or something else

**Comment by ryanatkn** [comment]
could do this now or I'll do in a followup, these components should live in `$routes/` instead of `$lib`, similar change to Felt.social

**Comment by greatbacon** [comment]
> could do this now or I'll do in a followup, most of these components should live in `$routes/` instead of `$lib`, similar change to Felt.social. Similar to how `+layout.svelte` lives in `src/routes`, I think even the main Footer and Header belong there (made easier that we can import with `$routes/` now), except things like the GitHub logo that are more clearly library-like. Values would go in `$routes/about/`

Is the idea that if a component is only used in one place, it should live next to it's page in `routes`? Or is it that if it's in lib it could/should be upstreamed? I'm happy to do the work myself in this PR, I just need to know the guiding "why" behind it 😄

**Comment by ryanatkn** [comment]
Currently thinking of it like this: `$lib/` for things that are either:

- exported as part of the package's library on npm (or likely to be exported)
- likely to be upstreamed

`$routes/styles.css` makes sense because it's used by `$routes/+layout.svelte`, so now it's side-by-side. A lot of other things similarly belong at the root `$routes/` even if they're used by multiple routes.

**Comment by greatbacon** [comment]
> Currently thinking of it like this: `$lib/` for things that are either:
> 
> * exported as part of the package's library on npm (or likely to be exported)
> * likely to be upstreamed
> 
> Moving `$routes/styles.css` from `$lib` makes sense because it's used by `$routes/+layout.svelte`, so now it's side-by-side. A lot of other things similarly belong at the root `$routes/` even if they're used by multiple routes. `$routes/Footer.svelte` is a good example -- it makes sense for it to be adjacent to the main `+layout.svelte`, because it plays a similar role, the difference is that it's our custom component instead of the builtin. They're both "root layout components".
> 
> Currently `$lib/project.ts` is the only module there, and it makes sense there because we may publish it to an npm package, so we could, for example, display the values data on other projects. This addresses your first question right? We could publish this package to npm as `@feltjs/felt` and have the `project.values` available to felt.social and others.

Yes, this answers both of my questions, thank you!! 😄
